### DISCUSSION TITLE
j4ym0 / pia-qbittorrent-docker Slow speeds

### DISCUSSION THREAD

**Discussion by 702enigma** [qdm12/gluetun#1460] [Q&A] [answered]
Im using j4ym0
 PIA and Qbittorrent and cant seem to get over 10MBps no matter what settings I change. All my other connections are getting my full gig down.

**Comment by bnhf** [comment, marked as answer]
@702enigma 

You should probably ask the developer of that fork to open up Issues and Discussions for the fork. It's had more than 100 new commits, and is almost 1.5K commits behind this project.

![screenshot-github com-2023 03 19-15_05_32](https://user-images.githubusercontent.com/41088895/226209038-cdccdb7a-ab6a-4dfc-a512-0bb9c7fe98d4.png)

Alternatively, you could set up a stack using the latest Gluetun and Qbittorrent (though you might want to switch to Deluge based on issues people have been reporting with Qbit recently), and see how performance compares.
